### ISSUE TITLE
Link to Money docs is broken

### ISSUE THREAD

**Issue by PatheticMustan** [Multiverse/Multiverse-Core#2149]
https://dev.bukkit.org/projects/multiverse-core
has the link "Economy" which leads to
https://github.com/Multiverse/Multiverse-Core/wiki/Money
which is a page that doesn't exist. I think you meant for the link to lead to
https://github.com/Multiverse/Multiverse-Core/wiki/Currency

**Comment by Jadiction** [comment]
Yeah, I've changed it to the correct link in a revision, just need Dumptruckman to save the changes.
